### PR TITLE
Added TC6: UsingInspector Note

### PR DESCRIPTION
**Pull request by igorkorsukov**
https://user-images.githubusercontent.com/3818029/140076322-5f9112f0-976c-4abf-a4b1-1369e5c6a1de.mp4

### CODE REVIEW

File: src/autobot/autobottypes.h [hunk 1]
```diff
@@ -55,7 +55,7 @@ using Scripts = std::vector<Script>;
 
 // --- TestCase ---
 //! NOTE Scripts with test cases must have a global variable with the name specified in the constant
-static const char* TESTCASE_JS_GLOBALNAME("testCase");
+constexpr const char* TESTCASE_JS_GLOBALNAME = "testCase";
```
Comment (Jojo-Schmitz) [inline src/autobot/autobottypes.h:58]: Darm I was so close, tried `constexpr char* TESTCASE_JS_GLOBALNAME = "testCase";`

File: src/inspector/view/qml/MuseScore/Inspector/notation/notes/StemSettings.qml [hunk 2]
```diff
@@ -114,7 +114,7 @@ FocusableItem {
                 delegate: FlatRadioButton {
                     height: 70
 
-                    navigation.name: "FlagStyleGroup"
+                    navigation.name: "FlagStyle"+modelData.text
```
Comment (Jojo-Schmitz) [inline src/inspector/view/qml/MuseScore/Inspector/notation/notes/StemSettings.qml:117]: no spaces around the `+`?

File: src/inspector/view/qml/MuseScore/Inspector/notation/notes/internal/BeamTypeSelector.qml [hunk 3]
```diff
@@ -78,7 +78,7 @@ InspectorPropertyView {
                 hint: modelData["hint"] ?? ""
                 background.radius: gridView.cellRadius
 
-                navigation.name: hint
+                navigation.name: "BeamType"+hint
```
[the remark this answers is not in the document]
Comment (igorkorsukov) [reply src/inspector/view/qml/MuseScore/Inspector/notation/notes/internal/BeamTypeSelector.qml:81]: No, this is not for the user, this is for debugging and autotests

[the remark this answers is not in the document]
Comment (igorkorsukov) [reply src/inspector/view/qml/MuseScore/Inspector/notation/notes/internal/BeamTypeSelector.qml]: fixed, added spaces

File: src/inspector/view/qml/MuseScore/Inspector/notation/notes/StemSettings.qml (hunk 2 above)
Comment (igorkorsukov) [reply src/inspector/view/qml/MuseScore/Inspector/notation/notes/StemSettings.qml]: fixed, added spaces

**Review by Eism — APPROVED** [review]

**Pull request merged**